lua/telescope/builtin/init.lua

```diff
@@ -21,59 +30,252 @@ end
+--- Search for a string in your current working directory and get results live as you type (respecting .gitignore)
+---@param opts table: options to pass to the picker
+---@field grep_open_files boolean: restrict search to open files only, mutually exclusive with `search_dirs`
+---@field search_dirs table: directory/directories to search in, mutually exclusive with `grep_open_files`
 builtin.live_grep = require('telescope.builtin.files').live_grep
 
+--- Searches for the string under your cursor in your current working directory
+---@param opts table: options to pass to the picker
+---@field search string: the query to search
+---@field search_dirs table: directory/directories to search in
 builtin.grep_string = require('telescope.builtin.files').grep_string
+
+--- Lists files in your current working directory, respects .gitignore
+---@param opts table: options to pass to the picker
+---@field find_command table: command line arguments for `find_files` to use for the search, overrides default config
+---@field follow boolean: if true, follows symlinks (i.e. uses `-L` flag for the `find` command)
+---@field hidden boolean: determines whether to show hidden files or not
+---@field search_dirs table: directory/directories to search in
 builtin.find_files = require('telescope.builtin.files').find_files
+
 builtin.fd = builtin.find_files
+
+--- Lists files and folders in your current working directory, open files, navigate your filesystem, and create new
+--- files and folders
+--- - Default keymaps:
+---   - <cr> type: opens the currently selected file, or navigates to the currently selected directory
+---   - <C-e> type: creates new file in current directory, creates new directory if the name contains a trailing '/'
+---     - Note: you can create files nested into several directories with <C-e>, i.e. `lua/telescope/init.lua` would
+---       create the file `init.lua` inside of `lua/telescope` and will create the necessary folders (similar to how
+---       `mkdir -p` would work) if they do not already exist
+---@param opts table: options to pass to the picker
+---@field search_dirs table: directory/directories to search in
 builtin.file_browser = require('telescope.builtin.files').file_browser
+
+--- Lists function names, variables, and other symbols from treesitter queries
+---@field show_line boolean: if true, shows matching lines
 builtin.treesitter = require('telescope.builtin.files').treesitter
+
+--- Live fuzzy search inside of the currently open buffer
+---@param opts table: options to pass to the picker
 builtin.current_buffer_fuzzy_find = require('telescope.builtin.files').current_buffer_fuzzy_find
+
+--- Lists tags in current directory with tag location file preview (users are required to run ctags -R to generate tags
+--- or update when introducing new changes)
+---@param opts table: options to pass to the picker
+---@field ctags_file string: specify a particular ctags file to use
 builtin.tags = require('telescope.builtin.files').tags
```

Comment:
There are a lot more "hidden" options which can be found in the entry_maker.
Here for example:
`show_line`, `hide_filename`